inbm/dispatcher-agent/tests/unit/test_dispatcher.py

```diff
@@ -101,7 +101,30 @@ def test_pre_check_pass(self,
     @patch('inbm_lib.mqttclient.mqtt.mqtt.Client.subscribe')
     @patch('dispatcher.dispatcher_class.Dispatcher._send_result')
     @patch('dispatcher.dispatcher_class.Dispatcher.invoke_workload_orchestration_check')
-    def test_aota_thread_start_called(self,
+    def test_aota_thread_start_called_without_signature(self,
+                                      mock_workload_orchestration_func: Any,
+                                      mock_send_result: Any,
+                                      m_sub: Any,
+                                      m_connect: Any,
+                                      m_thread_start: Any,
+                                      mock_logging: Any) -> None:
+        xml = '<?xml version="1.0" encoding="UTF-8"?>' \
+              '<manifest><type>ota</type><ota><header><id>sampleId</id><name>Sample AOTA</name><description>' \
+              'Sample AOTA manifest file</description><type>aota</type><repo>remote</repo>' \
+              '</header><type><aota name="sample.rpm"><cmd>load</cmd><app>docker</app><fetch>http://www.example.com/</fetch>' \
+              '<version>1.0</version><containerTag>defg</containerTag><signature>abcdefg</signature><sigversion>384</sigversion>' \
+              '</aota></type></ota></manifest>'
+        d = TestDispatcher._build_dispatcher()
+        d.do_install(xml=xml, schema_location=TEST_SCHEMA_LOCATION)
+        mock_workload_orchestration_func.assert_called()
+        assert m_thread_start.called
```

Comment:
**Suggestion:** The test method 'test_aota_thread_start_called_without_signature' should verify the actual call to 'm_thread_start.start' rather than just asserting it was called. Use 'm_thread_start.start.assert_called_once_with()' with the expected arguments to ensure that the thread is started with the correct parameters. [best practice]
**NOTE:** Ensure that the suggested code is comprehensive. This serves as an example of the code you could use. The user should verify it after accepting the suggestion.
```suggestion
        m_thread_start.start.assert_called_once_with()
```

Reply:
thread start doesn't have any args here